## Tools

* BWA v0.7.9
* Samtools v0.1.19
* Pilon v1.23

## Running

We ran successively four iterations resulting in no snp change.


[`bash run_pilon_round1.sh`](https://github.com/Yedomon/Genome_Assembly_Fusarium_oxysporum_f.sp._sesami/blob/master/03.Assembly_polishing/run_pilon_round1.sh)

[`bash run_pilon_round2.sh`](https://github.com/Yedomon/Genome_Assembly_Fusarium_oxysporum_f.sp._sesami/blob/master/03.Assembly_polishing/run_pilon_round2.sh)


[`bash run_pilon_round3.sh`](https://github.com/Yedomon/Genome_Assembly_Fusarium_oxysporum_f.sp._sesami/blob/master/03.Assembly_polishing/run_pilon_round3.sh)


[`bash run_pilon_round4.sh`](https://github.com/Yedomon/Genome_Assembly_Fusarium_oxysporum_f.sp._sesami/blob/master/03.Assembly_polishing/run_pilon_round4.sh)


### References
* Bruce J, Walker BJ, Abeel T, Shea T, Priest M, Abouelliel A, Sakthikumar S, Cuomo CA, Zeng Q, Wortman J, Young SK, Earl AM (2014) Pilon: An Integrated Tool for Comprehensive Microbial Variant Detection and Genome Assembly Improvement. PLoS ONE 9(11): e112963. [DOI](https://doi.org/10.1371/journal.pone.0112963)
* Li H, Durbin R (2009) Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25(14):1754–1760. [DOI](https://doi.org/10.1093/bioinformatics/btp324)
* Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R, 1000 Genome Project Data Processing Subgroup (2009) The Sequence Alignment/Map format and SAMtools. Bioinformatics 25(16):2078–2079. [DOI](https://doi.org/10.1093/bioinformatics/btp352)
